app/services/container_dashboard_service.rb

```diff
@@ -160,4 +161,43 @@ def heatmaps
       }
     }
   end
+
+  def node_utilization
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    metrics = VimPerformanceDaily.find_entries({:tz => @controller.current_user.get_timezone})
+    metrics = metrics.where(:resource_type => 'ExtManagementSystem', :resource_id => resource_ids)
```

Comment:
Unbounded? Don't you want to limit this to a certain timestamp? (last 30 days, etc.) Isn't this going to dump all the data since when you added the provider?